src/modules/services_manager_service.rb

```diff
@@ -150,48 +164,33 @@ def reset
       true
     end
 
-    # Returns only enabled services, the rest is expected to be disabled
+    # Returns services to be exported to AutoYast profile
+    #
+    # @return [Hash{String => Array<String>}]
     def export
-      enabled_services = services.select do |service_name, properties|
-        enabled(service_name) && properties[:loaded] && can_be_enabled(service_name)
-      end
+      enabled_services  = exportable_enabled_services.keys | ServicesProposal.enabled_services
+      disabled_services = exportable_disabled_services.keys | ServicesProposal.disabled_services
 
-      # Only services modifed by the user to be disabled are exported
-      # to AutoYast profile, untouched services are not exported
-      disabled_services = services.select do |service_name, properties|
-        !enabled(service_name) && properties[:modified]
-      end
-
-      export_enable = enabled_services.keys | ServicesProposal.enabled_services
-      export_disable = disabled_services.keys | ServicesProposal.disabled_services
-
-      log.info "Export: enabled services: #{export_enable}, disabled services: #{export_disable}"
+      log.info "Export: enabled services: #{enabled_services}, disabled services: #{disabled_services}"
 
-      {
-        'enable' => export_enable,
-        'disable' => export_disable,
-      }
+      { "enable" => enabled_services, "disable" => disabled_services }
     end
 
     def import(profile)
       log.info "List of services from autoyast profile: #{profile.services.map(&:name)}"
-      non_existent_services = []
 
-      profile.services.each do |service|
-        case service.status
-        when 'enable'
-          exists?(service.name) ? enable(service.name) : non_existent_services << service.name
-        when 'disable'
-          exists?(service.name) ? disable(service.name) : non_existent_services << service.name
-        else
-          log.error("Unknown status '#{service.status}' for service '#{service.name}'")
-        end
-      end
+      known_services = profile.services.select { |service| exists?(service.name) }
+      unknown_services = profile.services - known_services
```

Comment:
you can do it in one step: `known_services, unknown_service = profile.services.partition { |service| exists?(service.name) }`

Reply:
Nice! Thank you :)